<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="notifications_add">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:#a82b2b;
    border-style: outset;
    border-width: 4px;
    border-radius: 3px;
    border-color: #FFFFFF;
    font: bold 18px;
    padding: 6px;
    min-width: 10em;


}


QPushButton:hover{
	background-color: #c51a1a;
    border-style: outset;
    border-width: 2px;
    border-radius: 1px;
    border-color: beige;
    font: bold 22px;
    min-width: 10em;
    padding: 6px;
}</string>
         </property>
         <property name="text">
          <string>
Добавление уведомления
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="layoutWithServers"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-21</y>
      <width>1231</width>
      <height>961</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.318182 rgba(48, 48, 48, 255), stop:0.534091 rgba(63, 69, 74, 255), stop:0.755682 rgba(48, 48, 48, 255), stop:0.965909 rgba(0, 0, 0, 255));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>меню</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>соханить данные</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action_5"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="text">
    <string>С сохранением данных</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>сохранить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>насторойки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
